### PR TITLE
Allow tooltips for UiTab

### PR DESCRIPTION
**Pull request by kilobyte2007**
It is nice to have tooltips when the type of the tab is "icon".

### CODE REVIEW

**Comment by JosephusPaye** [comment]
Looks good! However I wasn't able to test locally (was your fork is giving me a 404). Perhaps the close and reopen broke something?

**Comment by kilobyte2007** [comment]
Fixed now, please try again!

File: docs-src/pages/UiTabs.vue [hunk 1]
```diff
@@ -73,6 +73,18 @@
                 </ui-tab>
             </ui-tabs>
 
+            <h4 class="page__demo-title">Tabs with tooltips</h4>
+
+            <ui-tabs type="icon">
+                <ui-tab :key="tab.title" v-for="tab in demoTabs" :tooltip="tab.title">
+                    <template #icon>
+                        <ui-icon :icon="tab.icon"></ui-icon>
+                    </template>
```
Comment (JosephusPaye) [inline docs-src/pages/UiTabs.vue:82]: In my local testing, I could only get this to work when changed to
```suggestion
                    <ui-icon slot="icon" :icon="tab.icon"></ui-icon>
```

**Review by JosephusPaye — CHANGES_REQUESTED** [review]
One thing I noticed when testing locally...

File: docs-src/pages/UiTabs.vue (hunk 1 above)
Comment (JosephusPaye) [reply docs-src/pages/UiTabs.vue:82]: Looks like your fork is still on Vue 2, that's why it wasn't working. It's fine when merged with `next` which has Vue 3.

**Review by JosephusPaye — APPROVED** [review]
LGTM, thanks!

**Pull request merged**